samples/project-geometry/src/main/AndroidManifest.xml

```diff
@@ -5,8 +5,7 @@
 
     <application><activity
             android:exported="true"
-            android:name=".MainActivity"
-            android:label="@string/project_geometry_app_name">
```

Comment:
Just come upon this statement in [Kotlin Samples Contribution Guide](https://devtopia.esri.com/runtime/common-samples/wiki/Kotlin-Samples-Repo%3A-A-Contribution-Guide):

> The sample viewer will only correctly name each individual sample if `android:label="@string/app_name"` is contained within the activity. So please make sure that in addition to any other lines, your activity contains the label. For example:
```
        <activity
            android:name=".MainActivity"
            android:label="@string/app_name">
```

Reply:
That's interesting. The sample seems to be named correctly in the sample viewer without it, and I checked a few other compose based samples and they seem to lack it as well. I wonder if perhaps that document is outdated or if there is some other mechanism at play that I've missed?

Reply:
Seems like this is indeed [the old approach](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/pull/296)